qdrant-landing/content/documentation/tutorials/async-api.md

```diff
@@ -0,0 +1,362 @@
+
+#### Asynchronous semantic search
+
+Search operation requires a query vector and a collection name. We can also specify the number of results we want to retrieve and some other
+parameters, but let's take one step at a time. The minimal example may look like this:
+
+```python
+response = await client.async_grpc_points.Search(
+    grpc.SearchPoints(
+        collection_name="my_collection",
+        vector=np.random.rand(100).tolist(),
+        limit=1,
+    )
+)
+
+print(response)
+```
+
+The response is going to contain just the point id and similarity score. For example, that might be the output of the code above:
+
+```text
+result {
+  id {
+    num: 1
+  }
+  score: 0.786471784
+  version: 1
+}
+time: 0.000456134
+```
+
+##### Returning vectors and payloads
+
+If you need the vectors and payloads of the points, you can specify it in the request:
+
+```python
+response = await client.async_grpc_points.Search(
+    grpc.SearchPoints(
+        collection_name="my_collection",
+        vector=np.random.rand(100).tolist(),
+        limit=1,
+        with_payload=grpc.WithPayloadSelector(enable=True),
+        with_vectors=grpc.WithVectorsSelector(enable=True),
+    )
+)
+```
+
+Right now, our response is going to have both vector and payload.
+
+##### Applying filters
+
+If you ever used SQL, you are probably familiar with the `WHERE` clause. It allows you to filter the results of the query. Qdrant has a similar
+functionality. Let's say we want to find all the points similar to given vector, but with `int_param` equal to 32. We can do it like this:
+
+```python
+response = await client.async_grpc_points.Search(
+    grpc.SearchPoints(
+        collection_name="my_collection",
+        vector=np.random.rand(100).tolist(),
+        limit=1,
+        with_payload=grpc.WithPayloadSelector(enable=True),
+        filter=grpc.Filter(
+            must=[
+                grpc.Condition(
+                    field=grpc.FieldCondition(
+                        key="int_param",
+                        match=grpc.Match(integer=32)
+                    )
+                )
+            ]
+        )
+    )
+)
+```
+
+We can build even more sophisticated filters with the `must`, `should` and `must_not` clauses. You can find more details [in the 
+documentation](https://qdrant.tech/documentation/concepts/filtering/#filtering).
+
+Analogous to search, we can also [scroll our collection](https://qdrant.tech/documentation/concepts/points/#scroll-points) to retrieve all the 
+points or use the [recommendation API](https://qdrant.tech/documentation/concepts/search/#recommendation-api) to find points similar to positive
+and dissimilar to negative examples.
+
+## Support of Qdrant async API in Python libraries
+
+There is plenty of Python libraries that Qdrant integrates with. Up till now, only [Langchain]() provides async API support. 
+Qdrant is the only vector database with full coverage of async API in Langchain. The documentation [describes how to use 
```

Comment:
Should we move this whole paragraph to the beginning - really drives home why this is an important tutorial

Reply:
I see. I think I was just trying to find a more powerful opening for why people should care. 
Maybe something in the first paragraph like?:

With Qdrant's async API, we are the only vector database to full support Langchain's async API.

Reply:
I want to avoid mentioning Langchain, as that might be a separate tutorial. Still, we can think about a better opening and maybe refer to well-known tools that support asynchronous programming (such as FastAPI) or other articles describing the pros and cons of async API.